--- /dev/null
+++ xserver-xorg-video-cirrus/src/cirrus.ids
@@ -0,0 +1,11 @@
+101300A0
+101300A4
+101300A8
+101300AC
+101300B8
+101300BC
+101300D0
+101300D4
+101300D5
+101300D6
+10130038
--- xserver-xorg-video-cirrus.orig/src/Makefile.am
+++ xserver-xorg-video-cirrus/src/Makefile.am
@@ -57,3 +57,6 @@
          lg_i2c.c \
          lg_xaa.c \
          lg_xaa.h
+
+pcidatadir = $(datadir)/xserver-xorg/pci
+pcidata_DATA = cirrus.ids
